app/src/components/common/TmSessionImportName.vue

```diff
@@ -100,18 +106,32 @@ export default {
       },
     },
   },
-  async created() {
-    this.importedAddress = await this.$store.dispatch(`getAddressFromSeed`, {
-      seedPhrase: this.$store.state.recover.seed,
-      network: this.network,
-    })
+  mounted() {
+    this.created()
   },
   methods: {
     onSubmit() {
       this.$v.$touch()
       if (this.$v.name.$invalid) return
       this.$router.push("/recover/password")
     },
+    async created(retry = false) {
+      if (retry) this.attempt++
+      const createAddressResponse = await this.$store.dispatch(
+        `getAddressFromSeed`,
+        {
+          seedPhrase: this.$store.state.recover.seed,
+          network: this.network,
+          // accountType: undefined,
+          attempt: this.attempt,
```

Comment:
I would go for passing `accountType` instead of `attempt` to be more explicit

Reply:
Well, that is a possible refactor, and then delete `getWalletWithRetry` altogether. Sounds fair